### PR TITLE
Send web3 RPC requests over RN module

### PR DESCRIPTION
**Pull request by rasom**
also solves #805

### CODE REVIEW

**Comment by rasom** [comment]
Two important things for testing here on Android:
1) check if node's RPC server is unavailable when app is in background
2) check that if app is entered background with open `/browse` dapp continue to work as expected after resume

**Comment by annadanchenko** [comment]
On Android, resume after going into background with Etherplay opened cause "Connecting" for long time.
Steps:
- run app and open Etherplay
- in Etherplay tap on ropsten to load screen with Play button. 
- If it says "syncing" wait will it's done
- Move app to background
- Move app back 
As a result, red "Connecting" message is shown. It stays for more than 5 min, so it looks like resume after going into background interrupts this app

**Comment by rasom** [comment]
@annadanchenko, fixed http://artifacts.status.im:8081/artifactory/pull-requests/im.status.ethereum-834d50.apk

**Comment by annadanchenko** [comment]
Looks OK.
- tested on iOS:
    - can sync
    - can send/recieve messages
- tested Android:
    - can sync
    - can send/recieve messages
    - open localhost:8545 in browser while Status is on background - site can't be reached
    - can put Status in background while Etherplay is loaded and resume -> DApp can be used 
    - can run Status. lock/unlock phone login and use app

File: package.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "react-native-tcp": "^3.2.1",
     "react-native-udp": "^2.0.0",
     "react-native-vector-icons": "^4.0.1",
-    "react-native-webview-bridge": "github:status-im/react-native-webview-bridge#react-native-0.40",
+    "react-native-webview-bridge": "github:status-im/react-native-webview-bridge#0.33.12",
```
Comment (alwx) [inline package.json]: you probably already did it, but you also have to rebase 0.33.12 branch :)

Comment (rasom) [inline package.json]: I merged `react-native-0.40` into `master`, so `0.33.12` contains it

**Pull request merged**